pkg/await/apps_deployment.go

```diff
@@ -164,15 +167,24 @@ func (dia *deploymentInitAwaiter) Await() error {
 	}
 	defer podWatcher.Stop()
 
+	// Create PersistentVolumeClaims watcher.
+	pvcWatcher, err := pvcClient.Watch(metav1.ListOptions{})
+	if err != nil {
+		return errors.Wrapf(err,
+			"Could not create watcher for PersistentVolumeClaims objects associated with Deployment '%s'",
```

Comment:
FYI, you can use `%q` to produce a quoted string rather than using `'%s'`

Reply:
Done! Thanks for the helpful tip